SwaggerGen/Swagger/Type/AbstractType.php

```diff
@@ -20,6 +20,35 @@
 	const REGEX_DEFAULT = '(?:=(.+))?';
 	const REGEX_END = '$/i';
 
+	private static $classTypes = array(
+		'integer' => 'Integer',
+		'int' => 'Integer',
+		'int32' => 'Integer',
+		'int64' => 'Integer',
+		'long' => 'Integer',
+		'float' => 'Number',
+		'double' => 'Number',
+		'string' => 'String',
+		'uuid' => 'StringUuid',
+		'byte' => 'String',
+		'binary' => 'String',
+		'password' => 'String',
+		'enum' => 'String',
+		'boolean' => 'Boolean',
+		'bool' => 'Boolean',
+		'array' => 'Array',
+		'csv' => 'Array',
+		'ssv' => 'Array',
+		'tsv' => 'Array',
+		'pipes' => 'Array',
+		'date' => 'Date',
+		'datetime' => 'Date',
+		'date-time' => 'Date',
+		'object' => 'Object',
+		'allof' => 'AllOf',
+	);
```

Comment:
Please add explicit `refobject`. See a few comments below.

Reply:
Sure. The list I copy/pasted this from (in `Schema` class, IIRC) must have been missing this too.

Reply:
It wasn't so much missing as merely unnecessary. In the latter case, you can't unambiguously imply a ReferenceObject through syntax. It doesn't hurt to offer the explicit `refobject` in the former situations though.